packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1390,11 +1391,11 @@ export default class Meeting extends StatelessWebexPlugin {
     this.remoteMediaManager = null;
 
     this.localAudioStreamMuteStateHandler = () => {
-      this.audio.handleLocalStreamMuteStateChange(this);
+      this.audio?.handleLocalStreamMuteStateChange(this);
```

Comment:
Just a question. Why have the audio and video objects become optional?

Reply:
We close the media connection and do cleanup. There is a possibility that the audio or video stream is already cleaned up by the time the event is emitted. That is why we have a null check here